### PR TITLE
Update history of v1.3.0-beta2

### PR DESCRIPTION
**Pull request by emagnier**
We are closed to a `v1.3.0-rc` and a public release!

@tukusejssirs, do you know what is the status on these 2, if there anything to do yet:
- https://github.com/romcal/romcal/issues/10 bug: duplicate celebrations
- https://github.com/romcal/romcal/issues/25 bug: change date of the feast of Our Lord Jesus Christ, The Eternal High Priest

There is also theses 2 PRs that might need a rebase if it's relevant:
- https://github.com/romcal/romcal/pull/46 Add missing feasts to `skSk.mjs`
- https://github.com/romcal/romcal/pull/64 Backport Slovak celebrations and localisation

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -1,13 +1,17 @@
...
 * Locale files have been renamed to kebab-case with capitalized regions to follow the IETF Language Codes standards and be close to the general conventions in software development for i18n.
+* On locale files, the `general` and `national` parts are now merged in a new `sanctoral` part. This will avoid potential duplicated or missing keys between these 2 parts, particularly when a celebration is move from the general to the national calendar. Also, keys in celebrations and the new sanctoral have been sorted alphabetically. ([#97](https://github.com/romcal/romcal/pull/97))
+* Romcal in Italian ! New `calendars/italy.js` and `locales/it.js` added.
+* Potential breaking changes:
+    - The `WEKDAY` type is renamed to `FERIA` ([#88](https://github.com/romcal/romcal/issues/88))
```
Comment (tukusejssirs) [inline HISTORY.md]: `WEKDAY` → `WEEKDAY`

File: HISTORY.md [hunk 2]
```diff
@@ -1,13 +1,17 @@
...
 * Locales management and pickup in romcal have been refactored, and it now behaves like Moment.js (used by romcal) or the general approach for i18n in app development: locale containing region like `fr-CA` will gracefully fall back to `fr` if a localisation key isn't in `fr-CA` or if `fr-CA.js` doesn't exits in the `src/locales` directory. In the end, it always falls back to `en`, which is the default language in romcal. Full explanations have been added to the `README.md`.
 * Locale files have been renamed to kebab-case with capitalized regions to follow the IETF Language Codes standards and be close to the general conventions in software development for i18n.
+* On locale files, the `general` and `national` parts are now merged in a new `sanctoral` part. This will avoid potential duplicated or missing keys between these 2 parts, particularly when a celebration is move from the general to the national calendar. Also, keys in celebrations and the new sanctoral have been sorted alphabetically. ([#97](https://github.com/romcal/romcal/pull/97))
```
Comment (tukusejssirs) [inline HISTORY.md]: I would say _in locale files_.

File: HISTORY.md (hunk 1 above)
Comment (emagnier) [reply HISTORY.md]: Fixed. Thanks!

File: HISTORY.md (hunk 2 above)
Comment (emagnier) [reply HISTORY.md]: Fixed. Thanks!

File: HISTORY.md [hunk 3]
```diff
@@ -1,13 +1,17 @@
...
 * Locales management and pickup in romcal have been refactored, and it now behaves like Moment.js (used by romcal) or the general approach for i18n in app development: locale containing region like `fr-CA` will gracefully fall back to `fr` if a localisation key isn't in `fr-CA` or if `fr-CA.js` doesn't exits in the `src/locales` directory. In the end, it always falls back to `en`, which is the default language in romcal. Full explanations have been added to the `README.md`.
 * Locale files have been renamed to kebab-case with capitalized regions to follow the IETF Language Codes standards and be close to the general conventions in software development for i18n.
+* In locale files, the `general` and `national` parts are now merged in a new `sanctoral` part. This will avoid potential duplicated or missing keys between these 2 parts, particularly when a celebration is move from the general to the national calendar. Also, keys in celebrations and the new sanctoral have been sorted alphabetically. ([#97](https://github.com/romcal/romcal/pull/97))
+* Romcal in Italian ! New `calendars/italy.js` and `locales/it.js` added.
```
Comment (tukusejssirs) [inline HISTORY.md]: I would reformulated this. Maybe just simply state that _Itallian translation has been added_. Also, you probably want to link the PR #94 and say that @gtortone has aided you. :smiley:

Comment (emagnier) [reply HISTORY.md]: Right, i forgot to add the link on that one!

**Comment by tukusejssirs** [comment]
> #10 bug: duplicate celebrations

I still need to check if they were all fixed.

> #25 bug: change date of the feast of Our Lord Jesus Christ, The Eternal High Priest

I have commented in that issue that it was not fixed properly.

> #46 Add missing feasts to skSk.mjs

I still have a couple of this to do.

> #64 Backport Slovak celebrations and localisation

Should we do this? I think when we release a `v1.3.0`, we will create a new stable branch and `master` will become obsolete. What do you think?

**Comment by emagnier** [comment]
When we are ready, `romcal-es6` will be merged to `master` and we will release `v1.3.0`. Then the `romcal-es6` branch will be deleted. So #64 isn't necessary.

**Pull request merged**